src/Shared/WebEncoders/WebEncoders.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.Extensions.Internal
 #endif
     static class WebEncoders
     {
-        private static readonly byte[] EmptyBytes = new byte[0];
+        private static readonly byte[] EmptyBytes = Array.Empty<byte>();
```

Comment:
Do we even need this field anymore? Can Array.Empty replace it everywhere?

Reply:
I did consider that, but wasn’t sure if there was a reason it was using the field rather than calling the method at runtime so left it. Happy to remove it and call Empty() at the call sites instead.

Reply:
The old one was caching the allocation, but I think Array.Empty does that internally.